mods/cnc/chrome/settings.yaml

```diff
@@ -585,6 +585,13 @@ Container@SETTINGS_PANEL:
 							Height: 20
 							Font: Regular
 							Text: Show Map Debug Messages
+						Checkbox@REPLAY_COMMANDS_CHECKBOX:
+							X: 310
+							Y: 220
+							Width: 300
+							Height: 20
+							Font: Regular
+							Text: Enable debug commands in replays
```

Comment:
This should use title case to match the other checkboxes: "Enable Debug Commands in Replays".